sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowUtils.scala

```diff
@@ -120,4 +121,19 @@ object ArrowUtils {
       StructField(field.getName, dt, field.isNullable)
     })
   }
+
+  /** Return Map with conf settings to be used in ArrowPythonRunner */
+  def getPythonRunnerConfMap(conf: SQLConf): Map[String, String] = {
+    val timeZoneConf = if (conf.pandasRespectSessionTimeZone) {
+      Seq(SQLConf.SESSION_LOCAL_TIMEZONE.key -> conf.sessionLocalTimeZone)
+    } else {
+      Nil
+    }
+    val pandasColsByPosition = if (conf.pandasGroupedMapAssignColumnssByPosition) {
```

Comment:
Can we do:
```
val pandasColByPosition = Seq(SQLConf.PANDAS_GROUPED_MAP_ASSIGN_COLUMNS_BY_POSITION.key -> conf.pandasGroupedMapAssignColumnssByPosition)
```

Reply:
I think it's better to just omit the config for the default case, that way it's easier to process in the worker.

Reply:
I am sorry can you explain why it's easier to process in the worker?

Isn't it just removing the default value here:
https://github.com/apache/spark/pull/21427/files#diff-d33eea00c68dfd120f4ceae6381f34cdR99

Also one thing is not great about omitting the conf for default case is that you need to put the default value in two places..(both python and java)